### PR TITLE
Update env 2023.01.12

### PR DESCRIPTION
**Pull request by dcherian**

### CODE REVIEW

**Comment by dcherian** [comment]
@scottyhq can you take a look here please? I can't get an updated environment to work

**Comment by scottyhq** [comment]
Well, that was some frustrating debugging. I'm not sure ultimate what what going on but I was noticing log messages with version mismatches, including some packages depending on py310 after bumping to py311, such as:
```
(condalock) ➜  conda git:(update-env) ✗ grep py310 *yml
environment.yml:  - llvmlite=0.39.1=py310h58363a5_1
environment.yml:  - unicodedata2=15.0.0=py310h5764c6d_0
environment.yml:  - numba=0.56.4=py310ha5257ce_0

Encountered problems while solving:
  - package libabseil-20220623.0-cxx17_h05df665_6 has constraint abseil-cpp 20220623.0* conflicting with abseil-cpp-20211102.0-h27087fc_1
  - package libgrpc-1.51.1-h30feacc_0 has constraint grpc-cpp 1.51.1* conflicting with grpc-cpp-1.47.0-hbd84cd8_1
```

This comment led me to try removing the `poetry` cache on the machine i'm using locally and that seemed to help
https://github.com/conda-incubator/conda-lock/issues/315#issuecomment-1382923291

File: conda/environment-unpinned.yml [hunk 1]
```diff
@@ -14,9 +14,10 @@ dependencies:
   - cartopy
   - cf_xarray
   - dask
+  - datashader
   - distributed
   - gcsfs
-  - geoviews
+  - geoviews-core
```
Comment (scottyhq) [inline conda/environment-unpinned.yml:20]: avoids bringing in `qt`

File: conda/environment-unpinned.yml [hunk 2]
```diff
@@ -14,9 +14,10 @@ dependencies:
   - cartopy
   - cf_xarray
   - dask
+  - datashader
```
Comment (scottyhq) [inline conda/environment-unpinned.yml:17]: required for `hvplot.image(rasterize=True)`, but brings in numba which currently requires <py310

**Comment by dcherian** [comment]
> This comment led me to try removing the poetry cache on the machine i'm using locally and that seemed to help

Thanks Scott. We should add this to the docs in the `conda/` folder.

**Pull request merged**